implementations/rust/vault/src/ffi/mod.rs

```diff
@@ -348,9 +348,10 @@ pub extern "C" fn ockam_vault_hkdf_sha256(
                 });
             }
 
+            // Hard code empty info string for now
```

Comment:
@mikelodder7 what's the eventual state this should be in? why an empty info string? 

adding more detail to this comment here would be helpful.

Reply:
Hardcoded to be empty for now because any changes to the C layer requires an API change. This change was necessary to implement Enrollment since the info string is not left blank for that protocol, but is blank for the XX pattern. If we agree to change the API, then this wouldn't be hardcoded but received from a parameter in the C API. Elixir and other consumers would be expected to pass the appropriate flag.

The other option is to not expose the vault directly since it may confuse users about what to pass here and I don't like the idea of yelling at consumers through comments. Instead the vault could be encapsulated in channels and key exchanges. Either way, I don't want to change the API until this decision is finalized.

Reply:
@mikelodder7 - ok, I meant that this detail should be added to the comment _in the source code_ :) can you push an update to include this?

Reply:
Sounds like a good `// TODO:`

Reply:
> yelling at consumers through comments. 

no one is yelling -- but this is too obvious of a comment to be helpful. simplifying the explanation above in a `TODO` is a reasonable expectation so that this isn't forgotten about.

Reply:
Add this to the comments as `TODO`